octobot_evaluators/channels/matrix.py

```diff
@@ -60,6 +61,7 @@ async def send(self,
                 "evaluator_type": evaluator_type,
                 "eval_note": eval_note,
                 "eval_note_type": eval_note_type,
+                "eval_time": eval_time,
```

Comment:
Do Matrix consumers need to have the `eval_time` ?

Reply:
Right, it's stored in the matrix anyway, maybe we should remove it

Reply:
Yes, I think we should remove it if most evaluators don't use `eval_time`.